protocol_doc/HP PageWide Pro 477dw MFP.md

```diff
@@ -0,0 +1,808 @@
+	<scan:ContrastSupport>
+		<scan:Min>0</scan:Min>
+		<scan:Max>2000</scan:Max>
+		<scan:Normal>1000</scan:Normal>
+		<scan:Step>1</scan:Step>
+	</scan:ContrastSupport>
+	<scan:ThresholdSupport>
+		<scan:Min>0</scan:Min>
+		<scan:Max>255</scan:Max>
+		<scan:Normal>128</scan:Normal>
+		<scan:Step>1</scan:Step>
+	</scan:ThresholdSupport>
+	<scan:eSCLConfigCap>
+		<scan:StateSupport>
+			<scan:State>disabled</scan:State>
+			<scan:State>enabled</scan:State>
+		</scan:StateSupport>
+	</scan:eSCLConfigCap>
+	<scan:JobSourceInfoSupport>true</scan:JobSourceInfoSupport>
+</scan:ScannerCapabilities>
+```
+
+### `GET /eSCL/ScannerStatus`
+
+Returns the current state of the scanner, including active and last job states.
+
+- 200 OK – Success
+- 301 Moved - The client is redirected to a secure connection.
+- 401 Unauthorized - The client is challenged for access credentials.
+- 500 Internal Server Error - Unknown internal error.
+- 503 Service Unavailable – Server busy. Retry late
+
+_Request_
+
+```http request
+GET /eSCL/ScannerStatus HTTP/1.1
+HOST: 192.168.0.20
+```
+
+_Response_
+
+```http request
+HTTP/1.1 200 OK
+Server: HP HTTP Server; HP PageWide Pro 477dw MFP - D3Q20B; Serial Number: CN136MX02P; Built: Wed Oct 13, 2021 07:50:14PM {MAVEDWPP1N001.2142A.00}
+Content-Type: text/xml
+Transfer-Encoding: chunked
+Content-Encoding: gzip
+Cache-Control: must-revalidate, max-age=0
+Pragma: no-cache
+
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- THIS DATA SUBJECT TO DISCLAIMER(S) INCLUDED WITH THE PRODUCT OF ORIGIN. -->
+<scan:ScannerStatus xmlns:scan="http://schemas.hp.com/imaging/escl/2011/05/03" xmlns:pwg="http://www.pwg.org/schemas/2010/12/sm" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://schemas.hp.com/imaging/escl/2011/05/03 ../../schemas/eSCL.xsd">
+	<pwg:Version>2.5</pwg:Version>
+	<pwg:State>Idle</pwg:State>
+	<scan:AdfState>ScannerAdfLoaded</scan:AdfState>
+	<scan:Jobs>
+		<scan:JobInfo>
+			<pwg:JobUri>/eSCL/ScanJobs/1c986213-09ff-1f09-a0d9-3822e23ba011</pwg:JobUri>
+			<pwg:JobUuid>1c986213-09ff-1f09-a0d9-3822e23ba011</pwg:JobUuid>
+			<scan:Age>93921</scan:Age>
+			<pwg:ImagesCompleted>9</pwg:ImagesCompleted>
+			<pwg:ImagesToTransfer>0</pwg:ImagesToTransfer>
+			<pwg:JobState>Completed</pwg:JobState>
+			<pwg:JobStateReasons>
+				<pwg:JobStateReason>JobCompletedSuccessfully</pwg:JobStateReason>
+			</pwg:JobStateReasons>
+		</scan:JobInfo>
+		<scan:JobInfo>
+			<pwg:JobUri>/eSCL/ScanJobs/1c9849e3-0997-1f09-b3ba-3822e23ba011</pwg:JobUri>
+			<pwg:JobUuid>1c9849e3-0997-1f09-b3ba-3822e23ba011</pwg:JobUuid>
+			<scan:Age>100247</scan:Age>
+			<pwg:ImagesCompleted>8</pwg:ImagesCompleted>
+			<pwg:ImagesToTransfer>0</pwg:ImagesToTransfer>
+			<pwg:JobState>Completed</pwg:JobState>
+			<pwg:JobStateReasons>
+				<pwg:JobStateReason>JobCompletedSuccessfully</pwg:JobStateReason>
+			</pwg:JobStateReasons>
+		</scan:JobInfo>
+	</scan:Jobs>
+</scan:ScannerStatus>
+
+```
+
+### `POST /eSCL/ScanJobs`
```

Comment:
Before this, I can't see what does trigger it

According to this page : https://support.hp.com/us-en/document/c05500578 I would expect something that tell the user as touched either `Send` or `Start Scan`

Do you know how the "Scan to Computer" tool knows that it has to request a scan ?